Extension/src/githubAPI.ts

```diff
@@ -191,7 +191,7 @@ export function getTargetBuild(builds: Build[], userVersion: PackageVersion, upd
     }
     const latestVersion: PackageVersion = new PackageVersion(builds[0].name);
```

Comment:
Can you rename this variable to `latestVersionOnline` so that it's clear what we're checking here?  I think it would be good to add a test for this case as well.

Reply:
new test added:
![image](https://user-images.githubusercontent.com/16143955/90059905-b7804d00-dc98-11ea-8ba3-024cc2ccefee.png)